_docs_integrate/data-model-overview.md

```diff
@@ -437,124 +437,124 @@ RequestItems can be sent inside of a [Request](#request) and specify what should
+| mustBeAccepted        | `boolean`                     | The mandatory `mustBeAccepted` property is used to differentiate between required and optional RequestItems within the Request. More specifically, this means that when a [Request is accepted]({% link _docs_use-cases/use-case-consumption-accept-incoming-request.md %}), depending on how the value of the <code>mustBeAccepted</code> property of a RequestItem is set, it is either necessary to accept it or it is possible to reject it.<br>{::nomarkdown}<ul><li>If <code>mustBeAccepted</code> is set to <code>true</code>, the peer cannot accept the Request without accepting this RequestItem. For example, some Attributes are mandatory for the business process and thus the respective RequestItems must be accepted.</li><li>If <code>mustBeAccepted</code> is set to <code>false</code>, the peer can accept the Request even though the RequestItem is rejected. For example, a consent to a newsletter is optional and thus does not need to be accepted, which is why <code>mustBeAccepted</code> would be set to <code>false</code> accordingly.</ul>{:/}If the RequestItem appears within the `items` property of a [RequestItemGroup](#requestitemgroup), the value of its `mustBeAccepted` property may have a slightly different effect depending on whether the value of the `mustBeAccepted` property of the RequestItemGroup is set to `true` or `false`. More details can be found in the description of the `mustBeAccepted` property of the [RequestItemGroup](#requestitemgroup). |
```

Comment:
> More specifically, this means that when a [Request is accepted]({% link _docs_use-cases/use-case-consumption-accept-incoming-request.md %}), depending on how the value of the <code>mustBeAccepted</code> property of a RequestItem is set, it is either necessary to accept it or it is possible to reject it.

Actually, I think this sentence can be removed completely, as it doesn't really add new information. But I don't mind if you'd like to keep it anyways. @britsta Decide how you prefer it. Same of course of the other RequestItems.

Reply:
I would leave it as it is for now, because it still contains the link to the "Accept incoming Request" use case, but I agree with you that the sentence can also be removed and would take this into account as soon as the documentation is rewritten because of the removal of the `mustBeAccepted` property of the RequestItemGroup. 🐸